woocommerce/woocommerce-android
# Issue/13127 add edit sites button

Closes: #13127 


### Description

Adds new `Edit Stores` button to site picker when the list displays more than 1 site. Additionally, adds a feature flag to avoid showing this new button in production builds. 

### Testing information
1. Log into the app using WP.com credentials with an account that has multiple stores
2. Go to the site picker
3. Check the new button `Edit Stores` is displayed once the store list is loaded

### The tests that have been performed

The above

### Images/gif


https://github.com/user-attachments/assets/49b82575-68d0-49ef-abba-df3612e56baa


## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
